.vscode/tasks.json

```diff
@@ -4,9 +4,12 @@
   "version": "2.0.0",
   "tasks": [
     {
-      "label": "Build glsp-client",
+      "label": "Build all",
```

Comment:
Should we call that `Build glsp-client and glsp-client example`?
Otherwise we'll end up with multiple Build all tasks in the VSCode workspace.

Reply:
Not sure if really necessary. In a VS Code workspace you always see the contributing directory name next to the task name
eg."Build all | glsp-client". 
Anyways, I will have a look at it once I updatre the VSCode workspace to conform to this changes and change it if necessary